pkg/scheduler/preemption/preemption.go

```diff
@@ -172,11 +184,15 @@ func (p *Preemptor) IssuePreemptions(ctx context.Context, preemptor *workload.In
 	workqueue.ParallelizeUntil(ctx, parallelPreemptions, len(targets), func(i int) {
 		target := targets[i]
 		if !meta.IsStatusConditionTrue(target.Obj.Status.Conditions, kueue.WorkloadEvicted) {
-			origin := "ClusterQueue"
-			reason := "InClusterQueue"
+			origin := ClusterQueueOrigin
+			reason := InClusterQueueReason
 			if cq.Name != target.ClusterQueue {
-				origin = "cohort"
-				reason = "InCohort"
+				origin = CohortOrigin
+				if reclamation {
+					reason = InCohortReclamationReason
+				} else {
+					reason = InCohortFairSharingReason
+				}
```

Comment:
nit: consider breaking this out into a function. something like:

origin, reason := getPreemptionReason(sameCq, borrowing bool)